File: c/10813/10813.c
#include <stdio.h>

int main(){
    int N,M,temp;
    int i,j;
    scanf("%d %d",&N, &M);
    int arr[N];
    for (int i=0;i<N;i++){
        arr[i]=i+1;
    }
    for (int x=0;x<M;x++){
        scanf("%d %d",&i, &j);
        temp = arr[i-1];
        arr[i-1] = arr[j-1];
        arr[j-1] = temp;
    }
    for (int x=0; x<N; x++){
        printf("%d ",arr[x]);
    }
    printf("\n");

    return 0;
}

File: python/10813/10813.py
import sys
input= sys.stdin.readline

N,M=map(int, input().split())
arr=[]

for i in range(N):
    arr.append(i+1)
for _ in range(M):
    i,j=map(int, input().split())
    temp=arr[i-1]
    arr[i-1]=arr[j-1]
    arr[j-1]=temp
print(*arr)